web/admin/src/views/LocationsListPage.vue

```diff
@@ -2,20 +2,33 @@
 import { ref, onBeforeMount } from 'vue'
 import { useAuthUserStore } from '@/stores/authUser'
 import { collection, getFirestore, query, where, doc, getDocs, addDoc, updateDoc, orderBy } from 'firebase/firestore'
+import SortableTableHeader from '@/components/SortableTableHeader.vue'
 
 const user = useAuthUserStore()
 const locations = ref()
+const sortBy = ref("name")
+const sortAsc = ref(true)
 
-onBeforeMount( async () => {
+const refreshList = async () => {
   const db = getFirestore()
-  const q = query(collection(db, "location"), orderBy('name'))
+  const q = query(collection(db, "location"), orderBy(sortBy.value, sortAsc.value ? "asc" : "desc"))
   const locRef = await getDocs(q)
   const locarray = []
   locRef.forEach((loc)=> {
     locarray.push({id:loc.id,...loc.data()})
   })
   locations.value = locarray
+}
+
+const sortList = param => {
+  sortAsc.value = sortBy.value === param ? !sortAsc.value : true
+  sortBy.value = param
 
+  refreshList()
+}
+
+onBeforeMount( async () => {
+  await refreshList();
```

Comment:
```suggestion
  await refreshList()
```
No need for semicolon at the end of the line.